webapi/_test_data/all_data/beol-data.ttl

```diff
@@ -1459,3 +1459,40 @@
 	knora-base:hasSubListNode <http://rdfh.ch/lists/0801/youth> , <http://rdfh.ch/lists/0801/personal_contact> , <http://rdfh.ch/lists/0801/correspondence> , <http://rdfh.ch/lists/0801/books> , <http://rdfh.ch/lists/0801/living_circumstances> , <http://rdfh.ch/lists/0801/miscellanea> ;
 	rdfs:label "Personal life"@en .
 
+<http://rdfh.ch/lists/0801/optics> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Optics" ;
+    knora-base:listNodePosition 0 ;
```

Comment:
Could you check the `knora-base:listNodePosition`? It is set to 0 for all new nodes.

Reply:
Thanks for bringing my attention to it. It is fixed now